### PR TITLE
refactor: MigratationDataOverview component

### PR DESCRIPTION
**Pull request by aaschlote**

### CODE REVIEW

**Review by pgurusinga — APPROVED** [review]

File: app/components/MigrationDataOverview.tsx [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 type MigrationDataProps = {
   readonly migrationData?: Record<string, unknown>;
   readonly translations: Translations;
 };
 
-const getMigrationValueTranslation = (
+const renderMigrationValue = (
   translations: Translations,
-  value: string,
+  value: unknown,
```
Comment (pgurusinga) [inline app/components/MigrationDataOverview.tsx:13]: how come this now an unknown?

File: app/components/MigrationDataOverview.tsx [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
   key: string,
 ) => {
-  const translation = translations[value];
+  if (typeof value === "object" && value !== null) {
+    return Object.entries(value).map(([_, subValue]) => (
+      <p key={subValue}>{lookupOrKey(subValue as string, translations)} </p>
```
Comment (pgurusinga) [inline app/components/MigrationDataOverview.tsx:18]: is this by accident with extra whitespace?
```suggestion
      <p key={subValue}>{lookupOrKey(subValue as string, translations)}</p>
```

**Review by pgurusinga — COMMENTED** [review]
sorry I approved wrong PR earlier :D

File: app/components/MigrationDataOverview.tsx (hunk 1 above)
Comment (aaschlote) [reply app/components/MigrationDataOverview.tsx]: It was defined this in the past, I changed to have a better type definition :)

**Pull request merged**